components/openindiana/bhyve-fw/Makefile

```diff
@@ -53,10 +55,11 @@ COMPONENT_BUILD_ENV += GAS=/usr/bin/gas
 COMPONENT_BUILD_ENV += GAR=/usr/bin/gar
 COMPONENT_BUILD_ENV += GLD=/usr/bin/gld
 COMPONENT_BUILD_ENV += GOBJCOPY=/usr/bin/gobjcopy
+COMPONENT_BUILD_ENV += PYTHON=/usr/bin/amd64/python2.7
 
 # Four variants of the bhyve firmware are built;
 # DEBUG and RELEASE, and with and without CSM.
-VARIANTS =	DEBUG DEBUG.csm RELEASE RELEASE.csm
+VARIANTS =	DEBUG RELEASE RELEASE.csm DEBUG.csm
```

Comment:
Is the order important here?  If yes, then please add a comment explaining it.  If not, then why you change it?